### ISSUE TITLE
Support visio(.vsdx) export

### ISSUE THREAD

**Issue by seabeam** [hediet/vscode-drawio#126]
Amazing project!
If you plan to support visio export please?

**Comment by heartacker** [comment]
BTW， if supportting vsdx file, it's better to support preview of VSdx file. thanks

**Comment by hediet** [comment]
> Amazing project!

Thank you! ;)

> If you plan to support visio export please?

I haven't planned that yet. Does drawio already support this?

**Comment by heartacker** [comment]
yes, drawio is supported. including prrerview and export vsdx file
